.gitignore

```
@@ -0,0 +1,7 @@
+
+modules/special-rules/.DS_Store
+modules/special-rules/.DS_Store
+modules/.DS_Store
+modules/special-rules/.DS_Store
+modules/special-rules/.DS_Store
+modules/.DS_Store
```

To deduplicate and match more paths:
```suggestion
.DS_Store
```
I believe that pattern will this will match and ignore this name anywhere in the local repo folder.
`**/.DS_Store` would do the same apparently, it's just more verbose.
@IJW-Wartrader Perhaps you could give this a try locally and check that it works before we commit it?

---

I'm pretty sure the suggestion will work, so I'll commit it to resolve this conversation.